test/assert.js

```diff
@@ -1461,13 +1460,13 @@ describe('assert', function () {
       assert.propertyVal(dummyObj, 'a', '2', 'blah');
     }, "blah: expected { a: '1' } to have property 'a' of '2', but got '1'");
 
-    expect(function () {
+    chai.expect(function () {
       assert.nestedProperty({a:1}, '{a:1}');
-    }).to.not.throw('the argument to `property` must be a string');
+    }).to.not.throw('the argument to property must be a string when using nested syntax');
 
-    expect(function () {
+    chai.expect(function () {
       assert.nestedProperty({a:1}, {'a':'1'});
-    }).to.throw('the argument to `property` must be a string');
+    }).to.throw('the argument to property must be a string when using nested syntax');
   });
 
```

Comment:
we are using `chai.expect` on line 18 too.

Reply:
It's shouldn't be wrong to use `expect` in this test, because here we are actually using `chai` to assert, we are not calling `assertions` and expecting them to work, we are testing that the assertion `throws`.

But I agree with @meeber that we must try and be able to use own `interface` to test it.

You can replace the ones you added on this PR with `assert.throws` and maybe remove the others in another PR.

Reply:
Sure @vieiralucas . I have replaced them and also surely will open a seperate PR in future to replace the remaining.